tests/arch/riscv/pmp/no-multithreading/testcase.yaml

```diff
@@ -0,0 +1,13 @@
+common:
+  platform_allow:
+    - qemu_riscv32
+    - qemu_riscv32e
+    - qemu_riscv64
+  filter: CONFIG_RISCV_PMP
+  ignore_faults: true
+
+tests:
+  arch.riscv.pmp.no-mt.isr-stack-guard:
+    extra_args: EXTRA_CFLAGS=-DPMP_TEST_FUNC_IDX=0
```

Comment:
could have used Kconfig instead?

Reply:
I think Kconfig option usually meant to allow user to tweak something.
Here it's already possible to select different tests by name, so I don't see benefits in using Kconfig.